D/DiffEqMonteCarlo/Compat.toml

```diff
@@ -5,15 +5,15 @@ RecipesBase = "0.1-0"
 ["0-0.6"]
 julia = "0.5-0"
 
-["0.10-0"]
+["0.10-0.14"]
 DiffEqBase = "3-5"
```

Comment:
Keep this?

Reply:
It shouldn't be capped above by 5 if that's what it's saying.

Reply:
Okay, but add something like
```toml
[compat]
DiffEqBase = "3, 4, 5, 6"
```
then? If it is compatible with all those major versions.